frontend/src/components/Job.vue

```diff
@@ -52,10 +68,10 @@ export default {
   color: white;
   font-weight: bold;
   font-size: 14px;
-  height: 100%;
   overflow: auto;
   display: inline-grid;
   align-items: center;
+  flex-grow: 1;
```

Comment:
Any specific reason to use **flex-grow**?

Reply:
This is to style the jobs in a way that the hide button is always at the bottom right corner. Without this, the hide button would be at a different position for each job, depending on how many lines the job name takes.